datamodel/create_roles.py

```diff
@@ -0,0 +1,237 @@
+#!/usr/bin/env python3
+
+import re
+from argparse import ArgumentParser, BooleanOptionalAction
+from typing import Optional
+
+from pgserviceparser import service_config
+
+try:
+    import psycopg
+except ImportError:
+    import psycopg2 as psycopg
+
+
+def get_db_identifier(pg_service: str, modelname: str):
+    dbname = service_config(pg_service).get("dbname")
+    db_identifier = re.sub(f"{modelname}_|teksi_", "", dbname)
```

Comment:
This would require some explanations on the different possibilities and ideally a small test covering this.